pkg/lib/machine-dialogs.scss

```diff
@@ -90,3 +90,11 @@
     visibility: hidden;
     height: 0px;
 }
+
+// When the dialog is ported to PF4, we can remove these lines.
+// (The element isn't currently imported, so its CSS variables aren't either. So we're defining what PF4 currently defines them as.)
+.pf-c-form__helper-text {
```

Comment:
You can import the CSS @import "@patternfly/patternfly/components/Form/form.css";

Reply:
Oh! That would be great. I was wondering about this. 👍🏼

Reply:
It doesn't work.

- When included in the SCSS file, it throws an error.
- When included in the JS file, it simply doesn't work.
- When included in the SCSS file as an SCSS file, instead of CSS, it also doesn't work.

When it doesn't work, the CSS is still included as:

```
.pf-c-form__helper-text {
    margin-top: var(--pf-c-form__helper-text--MarginTop);
    font-size: var(--pf-c-form__helper-text--FontSize);
    color: var(--pf-c-form__helper-text--Color);
}
```

(...As it was before including.)

But the variables still aren't assigned.

Reply:
I do see they're assigned in @patternfly/patternfly/components/Form/form.scss — but it somehow doesn't end up on the page when (supposedly) compiled in. 🤷🏼